### DISCUSSION TITLE
Should this be integrated with the dbatools project?

### DISCUSSION THREAD

**Discussion by Galaeci** [sqlcollaborative/dbaclone#195] [General]
It seems like this project would be a good fit within https://github.com/dataplat/dbatools and it could benefit from a greater pool of contributors there. 
I know merging projects is not a trivial task so it may not be a good fit at this point but maybe a future milestone for the roadmap.

**Comment by sanderstad** [comment]
Chrissy and I have talked about this in the past and we decided that merging this project into dbatools would not be a good fit. 
The reason for this is that this module requires several specifics like running as an admin.
Also, this project only supports Windows and dbatools is multi-platform.

**Reply by Galaeci** [reply]
Understood. Thanks for the quick reply and the awesome work you are doing here.
